esx_service/cli/vmdkops_admin_test.py

```diff
@@ -85,7 +85,7 @@ def test_tenant_create_missing_option_fails(self):
         self.assert_parse_error('tenant create')
 
     def test_tenant_rm(self):
-        args = self.parser.parse_args('tenant rm tenant1 --remove-volumes=True'.split())
+        args = self.parser.parse_args('tenant rm tenant1 --remove-volumes'.split())
```

Comment:
It should be 'tenant rm --name tenant1..." since "name" is not a positional argument.